api/CryptoKeyPair.json

```diff
@@ -11,6 +11,9 @@
           "chrome_android": {
             "version_added": "37"
           },
+          "deno": {
+            "version_added": false
```

Comment:
Some background which might be helpful for this and other Web APIs unsupported by Deno (e.g., `EventSource`):

On MDN we have some special handling for Web APIs for Node.js (and I expect, for Deno). Where there's no "nodejs" key, we don't show the column for Node; where there is, we show the column (even if it's `false` or `null`). This way, Node only appears on Web APIs that are relevant to non-browser usage.

(We don't do this for JavaScript features on Node. On JavaScript pages, we show that data is missing for Node.js.)

In other words, I'd only include Deno here if I wanted to highlight that it wasn't supported to developers using Deno. I suspect that's what you intended—given the support for other crypto features—but I thought I'd make it explicit.

Reply:
Thanks - that makes sense. My intention for everywhere I added `"version_added": false` was to convey the message that it is not _yet_ implemented (ie we do intend to implement it, and there are open issues). For APIs that we are not planning to add as of now (DOM or Service Worker) I simply didn't add a `"deno"` key to the file.